src/dswx_sar/filter_SAR.py

```diff
@@ -50,7 +50,13 @@ def compute_window_mean_std(arr, winsize):
     mean = masked_convolve2d(arr_masked, window, mode='same')
     c2 = masked_convolve2d(arr_masked*arr_masked, window, mode='same')
 
-    std = (c2 - mean * mean) ** .5
+    var = (c2 - mean * mean)
+
+    # The negative number in sqrt is replaced
+    # with the negligibly small number.
+    eps = 1e-10
```

Comment:
What do you think about importing `EPS` from `fuzzy_value_computation` so that `filter_sar.py` uses the same constant? Or you can do that in the other way around (i.e. define `EPS` in this code, and `fuzzy_value_computation.py` imports that)

Alternatively the constants can be defined in other code whose name looks more universal (e.g. `dswx_sar.py` or `dswx_sar_util.py`). What do you think?

Reply:
@seongsujeong Thank you for the suggestion. I add the constant class in `dswx_sar_util.py` and the others will import it.